portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,31 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <link href='https://fonts.googleapis.com/css?family=Martel Sans' rel='stylesheet'>
+    <style>
+    body {
+        font-family: 'Martel Sans';font-size: 22px;
+    }
```

Comment:
"Strictly keep structure (markup), presentation (styling), and behavior (scripting) apart." So, this should go in a CSS file.

https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns